src/Partition.ts

```diff
@@ -22,7 +23,7 @@ class Partition {
 		}
 	}
 
-	private async playSingle(playable: PlayableInterface): Promise<void> {
+	public async playSingle(playable: PlayableInterface): Promise<void> {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ Is this change necessary?

Reply:
Nope. I don't see any uses for it actually.